ingest/README.md

```diff
@@ -0,0 +1,37 @@
+# nextstrain.org/monkeypox/ingest
+
+This is the ingest pipeline for Monkeypox virus sequences.
+
+## Usage
+
+Fetch sequences with
+```
+nextstrain build . data/sequences.ndjson
+```
+
+Run the complete ingest pipeline with
+```
```

Comment:
This doesn't seem to run the ingest pipeline but the general Snakemake workflow instead, when my current working directory is at the top-level. Maybe we should add a sentence that one needs to be in the ingest folder?
Something like:
```sh
nextstrain build ingest/
```

Reply:
Ah yup, I was assuming if reading the `ingest/README` you were already in the ingest directory. 
I can add more language to clarify this!